### PR TITLE
Centralise IDE configurations

### PR DESCRIPTION
**Pull request by JackPGreen**
To improve the developer experience, I've created a place that common IDE configurations (code cleanups etc) can be shared.

Let me know your thoughts.

### CODE REVIEW

File: ide-configurations/netbeans/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+TODO Find a contributor
```
Comment (kwart) [inline ide-configurations/netbeans/README.md:1]: :grin: I guess @olukas will think about the contribution once he is done cleaning up his mailbox. :fire:

**Review by kwart — APPROVED** [review]
LGTM.  :+1:

**Review by vbekiaris — APPROVED** [review]
Nice job - it would be good to reference these config files from `CONTRIBUTING.md` which is the standard "guide" to contributors, so people can discover them.

**Comment by JackPGreen** [comment]
@olukas, @kwart said that you are using NetBeans and might have some configurations to share also?

**Comment by JackPGreen** [comment]
Following a discussion with @pveentjer, removed NetBeans - can be added separately later

**Pull request merged**